src/aap_eda/core/utils/logging_utils.py

```diff
@@ -25,3 +27,9 @@ def get_organization_name_from_data(data):
             else None
         )
     return org_name
+
+
+def feature_flag_analytics_disabled_logger():
```

Comment:
this function name suggests you're returning a logger, but it actually writes a message to the log.

Reply:
Function is removed now since it introduced a bug, reworked how this is done